src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import duke.exception.EmptyDescriptionException;
+import duke.exception.TaskNotFoundException;
+import duke.exception.WrongCommandException;
+
+import java.util.Scanner;
+
+import static duke.Parser.*;
```

Comment:
Instead of wildcard imports, declare your package imports explicitly, as with the [coding standard](https://se-education.org/guides/conventions/java/basic.html#package-and-import-statements).